README.md

```diff
@@ -156,34 +110,66 @@ nix-darwin.lib.darwinSystem {
 { pkgs, lib, inputs }:
 # inputs.self, inputs.nix-darwin, and inputs.nixpkgs can be accessed here
 ```
+</details>
 
-## Documentation
+<details>
+<summary>Channels</summary>
 
-Reference documentation of all the options is available [here](https://daiderd.com/nix-darwin/manual/index.html).
-This can also be accessed locally using `man 5 configuration.nix`.
+### Step 1. Creating `configuration.nix`
 
-`darwin-help` will open a HTML version of the manpage in the default browser.
+Copy the [simple](./modules/examples/simple.nix) example to `~/.config/nix-darwin/configuration.nix`.
 
-Furthermore there's `darwin-option` to introspect the settings of a system and its available options.
-> NOTE: `darwin-option` is only available to non-flake installations.
+### Step 2. Adding `nix-darwin` channel
 
+```bash
+nix-channel --add https://github.com/LnL7/nix-darwin/archive/master.tar.gz darwin
+nix-channel --update
 ```
-$ darwin-option nix.linux-builder.enable
-Value:
-true
 
-Default:
-false
+### Step 3. Installing `nix-darwin`
+
+Unlike NixOS, `nix-darwin` does not have an installer, you can just run `darwin-rebuild switch` to install nix-darwin. As `darwin-rebuild` won't be installed in your `PATH` yet, you can use the following command:
 
-Example:
-true
+```bash
+nix --extra-experimental-features "nix-command flakes" run nix-darwin -- switch -I darwin-config=$HOME/.config/nix-darwin/configuration.nix
```

Comment:
I think in the non‐flakes section we should offer a `nix-shell`‐based non‐flake command to do this.

Reply:
Updated 👍